### PR TITLE
Move to GitHub actions

### PR DESCRIPTION
**Pull request by MancunianSam**
* Add release versions job which publishes to GitHub actions.
* Add a terraform check job for the terraform

### CODE REVIEW

File: docs/output.html [hunk 1]
```diff
@@ -0,0 +1,136 @@
+<html>
```
Comment (TomJKing) [inline docs/output.html:1]: Should this be here? Looks like this a already generated output?

**Review by TomJKing — COMMENTED** [review]
Looks good. Just one minor question.

File: docs/output.html (hunk 1 above)
Comment (MancunianSam) [reply docs/output.html]: Ah no, the output file does get committed when the release versions job is run but it sits on the scripts-pages branch which is used by GitHub pages to show it so it shouldn't be on this branch. I've deleted it.

**Review by TomJKing — APPROVED** [review]
Looks good. Thanks.

**Review by TomJKing — APPROVED** [review]

**Pull request merged**